contracts/roles/ClaimManager.sol

```diff
@@ -117,8 +118,12 @@ contract ClaimManager is Initializable, UUPSUpgradeable, OwnableUpgradeable, EIP
       expiry,
       issuer
     ))));
+
+    require(proofHashes[proofHash] == false, "ClaimManager: The role has been issued already");
```

Comment:
It is possible to re-register a role, you just need a different proof, right? Like you could adjust the expiry date if you wanted to resubmit, for example.
```suggestion
    require(proofHashes[proofHash] == false, "ClaimManager: The proof has been submitted already");
```